catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/MigrateCommand.java

```diff
@@ -43,44 +47,66 @@
 import ddf.catalog.util.impl.ServiceComparator;
 
 @Command(scope = CatalogCommands.NAMESPACE, name = "migrate", description = "Migrates Metacards "
-        + "from a Configured External Provider into the Catalog.  The migrate command currently "
-        + "picks the first Catalog provider as the FROM and the second Catalog provider as the TO. "
-        + "Additional support will be added to specify which provider will be migrated TO and FROM "
-        + "in the future.")
+        + "from a Configured External Provider to another Provider.")
 public class MigrateCommand extends DuplicateCommands {
     private static final Logger LOGGER = LoggerFactory.getLogger(MigrateCommand.class);
 
+    @Option(name = "--list", required = false, aliases = {
+            "-list"}, multiValued = false, description = "Print a list of external providers before migrating.")
```

Comment:
Is this necessarily only printing external providers? Won't this `--list` option also print the default local provider? Similar comment for the `--from` and `--to` options.